src/simulators/statevector/qubitvector.hpp

```diff
@@ -238,6 +238,10 @@ class QubitVector {
   // The matrix is input as vector of the column-major vectorized N-qubit matrix.
   void apply_matrix(const reg_t &qubits, const cvector_t &mat);
 
+  // Apply N-qubit matrixes to the state vector.
+  // The matrixes are inputs as vectors of the column-major vectorized N-qubit matrixes.
+  void apply_matrixes(const std::vector<reg_t> &qubitss, const std::vector<cvector_t> &mats);
```

Comment:
Can you change this to `apply_matrix_sequence`: its a bit more descriptive of what the function does